documentation/book/security.adoc

```diff
@@ -31,31 +41,31 @@ NOTE: "certificates rotation" for generating new ones on their expiration will b
 
```

Comment:
Now that this document is less Kafka-specific I think we should change the section title from 'Listeners' to 'Kafka Listeners' for clarity